nginx/nginx.conf

```
@@ -1,20 +1,30 @@
-server {
+worker_processes auto;
+# Note: /var/log/nginx/error.log symlinks to stderr
+error_log  /var/log/nginx/error.log warn;
+pid        /tmp/nginx.pid;
 
-  listen 8080;
+events {
+    worker_connections 1024;
+}
 
-  
-  location ~ /paralog {
-    root   /usr/share/nginx/html;
-    index  index.html index.htm;
-    try_files $uri /paralog/index.html;
-  }
-  location = / {
-      return 301 /paralog/;
-  }
-  error_page   500 502 503 504  /50x.html;
+http {
+    include       /usr/local/nginx/conf/mime.types;
+    default_type  application/octet-stream;
 
-  location = /50x.html {
-    root   /usr/share/nginx/html;
-  }
+    # Custom log format
+    log_format main '$remote_addr - $remote_user [$time_local] '
+                    '"$request" $status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for"';
+    # Note: /var/log/nginx/paralog.log symlinks to stdout
+    access_log /var/log/nginx/paralog.log main;
```

This is wrong and due to  my misunderstanding. 

This actually needs to remain access.log main
```suggestion
    access_log /var/log/nginx/access.log main;
```
https://github.com/telicent-oss/catalog/blob/main/nginx/nginx.conf#L20